src/components/Explore/panels/histogram.ts

```diff
@@ -37,7 +37,9 @@ export function getHistogramVizPanel(scene: SceneObject, yBuckets: number[]) {
         const yTo = yBucketToDuration(args[0].y?.to || 0, yBuckets);
         newSelection.duration = { from: yFrom, to: yTo };
 
-        parent.setState({ selection: newSelection });
+        parent._urlSync.performBrowserHistoryAction(() => {
```

Comment:
I think this should be rather encapsulated in a separate function in the parent like [an example in docs](https://github.com/grafana/scenes/blob/main/docusaurus/docs/url-sync.md#sceneobjecturlsyncconfig). This way we will not need expose _urlSync of the parent which should remain private.